### ISSUE TITLE
Please expand the README.md

### ISSUE THREAD

**Issue by pvalena** [teemtee/tmt#2730]
Please, at least add quick howto like Install (or links to it). And/Or a short example for usage (not a big block of text like there is).

This is quite usual (and welcoming) for upstream projects to have this in the README.md :).

**Comment by BDFL669** [comment]
I'd be happy to help with this issue. Please assign it to me

**Comment by pvalena** [comment]
Glad to hear that! Please wait for the repo maintainers/owners to agree some changes are needed :).

**Comment by BDFL669** [comment]
Ok, thank you.

**Comment by LecrisUT** [comment]
Worth considering:
- https://github.com/teemtee/tmt/pull/2414 The contents of the README were moved recently to better support sphinx-specific decorators. But having a more shortened tl;dr version of that as a front-page is worth reconsidering
- synchronization of `README.md` with `index.md` ([example](https://github.com/scikit-build/scikit-build-core), see usage of `<!-- SPHINX-START -->`)
- Badges? Many cool ones from https://shields.io/badges
- What to include? Best discussed with the main developers, but some things I think would be nice
  - Purpose and goal of tmt. I view it as a tool to unify testing independent of CI environment
  - One simple `test.fmf` with the cli interface to trigger/CI environments that run it
  - Where to get in touch
  - Primary sponsors?

Anyway let's wait for more feedback from @lukaszachy @psss @thrix? Happy to review though.

**Comment by psss** [comment]
Yes, the original `README` content was moved because it has grown substantially and it did not make sense to maintain similar information on several places. I like the idea of providing a `tldr` version on the most essential commands. Perhaps, we could resolve this issue together with this one?

* #968 

The same content could be submitted to the `tldr` command repository. Plus, in the `README` we could include a link to the installation instructions to help users get quickly started?

**Comment by psss** [comment]
Summary from the triage meeting:

* Add a couple more useful links (e.g. first steps, guide, installation)

**Comment by martinhoyer** [comment]
hmm, PR in "Review" column, issue in "Implement" column. Who could have foreseen these things happening :)
